### ISSUE TITLE
Didn't find how to solve it

### ISSUE THREAD

**Issue by ReactorNefg** [DizzasTeR/VCMP-Lua#17]
Plugin error >> dlopen() 'plugins/LuaPlugin_x64.so' failed: plugins/LuaPlugin_x64.so: undefined symbol: mysql_use_result
Failed to load plugin: LuaPlugin_x64

OS: Linux debian-m-2vcpu-16gb-fra1-01 5.10.0-11-amd64 #1 SMP Debian 5.10.92-1 (2022-01-18) x86_64 GNU/Linux

**Comment by rasikhq** [comment]
Did you build the plugin yourself or got the pre-built binaries?

**Comment by ReactorNefg** [comment]
> Did you build the plugin yourself or got the pre-built binaries?

Downloaded from github

**Comment by rasikhq** [comment]
I should've specified this but the linux build works on ubuntu, I think you're better off with building the plugin on your own linux machine. If you have no experience doing this at all, I'll try to get your Debian builds

**Comment by ReactorNefg** [comment]
> I should've specified this but the linux build works on ubuntu, I think you're better off with building the plugin on your own linux machine. If you have no experience doing this at all, I'll try to get your Debian builds

It would be great. I tried to collect it did not work very well (

**Comment by ReactorNefg** [comment]
To check put Ubuntu nothing has changed.

Linux ubuntu-s-4vcpu-8gb-fra1-01 5.19.0-23-generic #24-Ubuntu SMP PREEMPT_DYNAMIC Fri Oct 14 15:39:57 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by rasikhq** [comment]
Yes I've tried it now, doesn't seem to work anymore. Seems like MySQL has changed. Had to happen eventually, this will take some time to setup again because at the moment I'm using really wacky ways to get mysql to work but it no longer does.

**Comment by rasikhq** [comment]
Okay I've wasted my weekend on this and I'd like to inform you guys with what I have in mind. Anything related to MySQL and MariaDB is being a real pain to setup with either premake or cmake and both of them are giving me weird issues.

Now to me the only viable is going to Postgres, which is pretty much better MySQL in every way and is getting popular every day, but before I make the leap, I do want to know if you guys (the users) are okay with this. Maybe look into postgres and give it a try and if you guys think you'll be able to switch over from mysql, great, otherwise it I don't know when or how to solve the build issues I'm getting with mysql/mariadb

**Comment by rasikhq** [comment]
I have a lead on solving this, will let you know soon @ReactorNefg

**Comment by ReactorNefg** [comment]
Ok

**Comment by Necroso** [comment]
> Okay I've wasted my weekend on this and I'd like to inform you guys with what I have in mind. Anything related to MySQL and MariaDB is being a real pain to setup with either premake or cmake and both of them are giving me weird issues.
> 
> Now to me the only viable is going to Postgres, which is pretty much better MySQL in every way and is getting popular every day, but before I make the leap, I do want to know if you guys (the users) are okay with this. Maybe look into postgres and give it a try and if you guys think you'll be able to switch over from mysql, great, otherwise it I don't know when or how to solve the build issues I'm getting with mysql/mariadb

Could you do it in PostgreSQL then? To solve this problem with Linux

**Comment by rasikhq** [comment]
Yes, but again, I didn't get much info on whether people who are using this are willing to switch over from mysql

**Comment by Necroso** [comment]
hey @rasikhq I replaced MySQL with PostgreSQL and confirmed it is working. The plugin now supports 32-bit Windows and is also running on Linux. If I am granted write access to the repository, I will publish these changes in an experimental branch.

**Comment by rasikhq** [comment]
@Necroso That's great, if you could create a PR that would be fantastic.

**Comment by Necroso** [comment]
@rasikhq done!
